frontend/apps/filemanager/filemanager.lua

```diff
@@ -471,8 +471,12 @@ end
 function FileManager:getSortingMenuTable()
     local fm = self
     local collates = {
-        strcoll = {_("by title"), _("Sort by title")},
-        access = {_("by date"), _("Sort by date")},
+        strcoll = {_("title"), _("Sort by title")},
+        access = {_("date read"), _("Sort by date of last read")},
+        change = {_("date added"), _("Sort by date of adding")},
```

Comment:
I think it'd sound better to say

Sort by last read date
Sort by date added
Sort by date modified